src/types.props.ts

```diff
@@ -59,10 +59,7 @@ export type FormikReimaginedProps<Values> = FormikReimaginedState<Values> &
 /**
  * Render properties of field array. Accessible through render and children.
  **/
-export interface FieldArrayRProps<Value> extends ArrayHelpers<Value> {
```

Comment:
In order to help typescript with type inference, let's remove one level of ....

Reply:
looks good, does this break compatability?

Reply:
Yes, though we haven't made a 1.0.0 release yet. But I can add a deprecated alias if you want.

Reply:
https://github.com/jaredpalmer/formik/pull/2237